app/views/gems/show.html.erb

```diff
@@ -26,6 +26,12 @@
   <% end %>
```

Comment:
Maybe we want to check if `@gem_adoption` has any errors and show an error message?